analyze_openapi_changes.py

```
@@ -0,0 +1,180 @@
+import argparse
+from datetime import datetime
+from deepdiff import DeepDiff
+from dotenv import load_dotenv
+from typing import Dict, Any, Optional, Tuple
+from openai import OpenAI
+
+load_dotenv()
+
+OPEN_API_KEY = os.getenv("OPEN_API_KEY")
+
+def load_json_file(file_path: str) -> Optional[Dict[str, Any]]:
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)